README.md

```diff
@@ -295,7 +295,12 @@ It is possible to use Iceberg in an incremental fashion, specifically two strate
     - `unique_key` **(required)**: columns that define a unique record in the source and target tables.
     - `incremental_predicates` (optional): SQL conditions that enable custom join clauses in the merge statement. This can be useful for improving performance via predicate pushdown on the target table.
     - `delete_condition` (optional): SQL condition used to identify records that should be deleted.
-      - `delete_condition` and `incremental_predicates` can include any column of the incremental table (`src`) or the final table (`target`). Column names must be prefixed by either `src` or `target` to prevent a `Column is ambiguous` error.
+    - `update_condition` (optional): SQL condition used to identify records that should be updated.
```

Comment:
nice, shall we add also to our docs repo?

Reply:
Sure, will add that. However we should find the solution to keep docs in this repo to simplify support process in long-term.

Reply:
docs are separated because we totally use another setup for docs deployment, and put all in one place is a bit hard. But happy to find a smoother solution - @Jrmyy and @jessedobbelaere some ideas?

Reply:
@nicor88 I believe this does not belongs to current PR, I've created extra issue for that https://github.com/dbt-athena/dbt-athena/issues/433